docs/src/pages/[platform]/components/view/examples/CustomAttributesExample.tsx

```diff
@@ -0,0 +1,5 @@
+import { View } from '@aws-amplify/ui-react';
+
+export const CustomAttributesExample = () => {
+  return <View as="progress" data-progress-bar max="100" value="75" />;
```

Comment:
Just a thought -- did you consider other example than `progress`? It might be just me, but I wasn't familiar with it so it took me a second to realize `max` and `value` are specific to progress.